src/common/Editor/Editor.jsx

```diff
@@ -1,5 +1,3 @@
-// Port from react-ace
```

Comment:
I suggest to have another ticket for that. It won't be a simple replace. Also whole saving mixin will need a update.